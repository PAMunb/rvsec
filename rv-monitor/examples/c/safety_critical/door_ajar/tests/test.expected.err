Test 1
Test 2
Attempting to start the engine with door open!
Test 1 ASPC
Test 2 ASPC
Attempting to start the engine with door open!
