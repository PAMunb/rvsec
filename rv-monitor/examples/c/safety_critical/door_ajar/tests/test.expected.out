Test 1
Door opened
Door closed
Attempting to start engine
Test 2
Door opened
Attempting to start engine
Door closed
Test 1 ASPC
Door opened
Door closed
Attempting to start engine
Test 2 ASPC
Door opened
Attempting to start engine
Door closed
